pippo-content-type-parent/pippo-xstream/src/main/java/ro/pippo/xstream/XstreamEngine.java

```diff
@@ -60,7 +64,6 @@ public String toString(Object object) {
         return xstream().toXML(object);
     }
 
-    @SuppressWarnings("unchecked")
```

Comment:
Without this annotation I retrieve a warning. The annotation was added by me in the previous commit.